src/rules/incognia-onboarding.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @title Incognia Onboarding Rule
```

Comment:
I believe this is for the Incognia Verification integration, correct? If so, can we use the same name here and slug version for the file, like `incognia-verification`?

Reply:
Sounds like this one is staying "Onboarding"